addons/purchase/static/src/js/purchase_portal_sidebar.js

```diff
@@ -139,12 +135,4 @@ publicWidget.registry.PurchasePortalSidebar = PortalSidebar.extend({
         });
         return rawText.join(" ");
     },
-    _onDecline: function (ev) {
-        const orderId = parseInt(ev.currentTarget.dataset.orderId);
-        this.orm.call("purchase.order", "decline_reception_mail", [orderId]);
-    },
-    _onAccept: function (ev) {
-        const orderId = parseInt(ev.currentTarget.dataset.orderId);
-        this.orm.call("purchase.order", "confirm_reception_mail", [orderId]);
```

Comment:
I believe `confirm_reception_mail` and `decline_reception_mail` were made public only for these calls. Maybe we can change them back to private?

Reply:
Good call! :slightly_smiling_face: